orm.go

```diff
@@ -760,13 +760,20 @@ func OptFieldTypeSet(cacheType CacheType, cacheSize int) FieldOption {
 func OptFieldTypeInt(limits ...int64) FieldOption {
 	min := int64(math.MinInt64)
 	max := int64(math.MaxInt64)
-	// note that we are not able to raise an error if len(limits) > 2
+
+	if len(limits) > 2 {
+		panic("error: OptFieldTypInt accepts at most 2 arguments")
+	}
 	if len(limits) > 0 {
 		min = limits[0]
 	}
 	if len(limits) > 1 {
 		max = limits[1]
 	}
+	if min > max {
+		panic("error: min cannot be greater than max")
+	}
+
```

Comment:
I don't actually think we should panic in this case. If there are more than two arguments then the actual structure of the code is wrong and it's unclear what we should do. But if Min is greater than Max that could be a situation where we've taken in external input, and we really don't want that to crash the whole program. In this case I think the best thing to do is send it to the server and let an error come back from there

Reply:
ok, updated

Reply:
did you forget to push?

Reply:
right, pushed.